pkg/users/accounts-list.js

```diff
@@ -42,18 +42,22 @@ import { SortByDirection } from "@patternfly/react-table";
 import { account_create_dialog } from "./account-create-dialog.js";
 import { delete_account_dialog } from "./delete-account-dialog.js";
 import { delete_group_dialog } from "./delete-group-dialog.js";
+import { group_create_dialog } from "./group-create-dialog.js";
 import { lockAccountDialog } from "./lock-account-dialog.js";
 import { logoutAccountDialog } from "./logout-account-dialog.js";
 
+import { usePageLocation } from "hooks";
+
 const _ = cockpit.gettext;
 
 const GroupActions = ({ group, accounts }) => {
     const [isKebabOpen, setKebabOpen] = useState(false);
 
-    if (!superuser.allowed)
-        return null;
-
     const actions = [
+        <DropdownItem key="edit-group"
+                      onClick={ev => { ev.preventDefault(); cockpit.location.go(["group", group.name]) }}>
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18136-20230116-100807-e09cfe13-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)